app/assets/javascripts/spree/frontend/spree_payu_integration.js

```diff
@@ -5,7 +5,7 @@
     },
 
     isPayuChoosen: function(e) {
-      return this.choosenPaymentMethodId(e) == this.payuPaymentMethodId();
+      return this.choosenPaymentMethodId(e) === this.payuPaymentMethodId();
```

Comment:
yup, had one enabled but commented out because of some upstream bug and forgot about it (pity), thanks for a reminder